cts/lab/CTStests.py

```diff
@@ -26,7 +26,7 @@
 from pacemaker import BuildOptions
 from pacemaker._cts.CTS import NodeStatus
 from pacemaker._cts.audits import AuditResource
-from pacemaker._cts.tests.base import CTSTest, RemoteDriver, SimulStartLite, SimulStopLite, StartTest, StopTest
+from pacemaker._cts.tests import CTSTest, RemoteDriver, SimulStartLite, SimulStopLite, StartTest, StopTest
```

Comment:
I'm usually not a fan of `import *` but I think you're going to need that here when we add a zillion more tests. Not necessary for this PR though